src/FxKit/Option/Option.Traverse.cs

```diff
@@ -19,7 +20,7 @@ public static partial class Option
     [GenerateTransformer]
     public static Validation<Option<TValid>, TInvalid> Traverse<T, TValid, TInvalid>(
         this Option<T> source,
-        Func<T, Validation<TValid, TInvalid>> selector)
+        [InstantHandle] Func<T, Validation<TValid, TInvalid>> selector)
```

Comment:
Always wondered if there was an attribute to tell the analyzer that the function is being invoked immediately - didn't know this was it, thanks!